engine/distributed.go

```diff
@@ -45,6 +45,10 @@ func (l remoteEngine) LabelSets() []labels.Labels {
 	return l.labelSets
 }
 
+func (l remoteEngine) PartitionLabelSets() []labels.Labels {
+	return l.labelSets
```

Comment:
This is a safe default. Engines who want to specify different partition labels can implement RemoteEngine differently.